koku/api/provider/test/tests_serializers.py

```diff
@@ -367,7 +367,9 @@ def test_create_gcp_provider(self):
             "name": "test_provider_one",
             "type": Provider.PROVIDER_GCP.lower(),
             "authentication": {"credentials": {"project_id": "gcp_project"}},
-            "billing_source": {"data_source": {"bucket": "test_bucket", "report_prefix": "precious"}},
+            "billing_source": {
+                "data_source": {"dataset": "test_dataset", "table_id": "test_table_id", "report_prefix": "precious"}
```

Comment:
I know the `report_prefix` was here before... But what what is the purpose for including it here?  Maybe that it's just ignored?

Reply:
I didn't know if the report prefix was used to help generate the path of where the csv file would go during the downloader. So, I went ahead and left it in here to make sure that it could still be used. However, while working on the downloader if we decide we don't need it then we can cut it out.